stdlib/public/core/Unicode.swift

```diff
@@ -63,13 +64,17 @@ public protocol UnicodeCodec {
   /// Because of buffering, it is impossible to find the corresponding position
   /// in the iterator for a given returned `UnicodeScalar` or an error.
   ///
-  /// - parameter next: An iterator over the code units to be decoded.
+  /// - Parameter next: A generator of code units to be decoded.  Repeated
```

Comment:
Actually I also found a few stray cases of generator in the rest of the stdlib, see #1705.